packages/admin-e2e-tests/src/utils/actions.ts

```diff
@@ -29,8 +34,9 @@ const verifyPublishAndTrash = async (
 	publishVerification: string,
 	trashVerification: string
 ) => {
+	const newOrder = new NewOrder( page );
 	// Wait for auto save
-	await page.waitFor( 2000 );
+	await newOrder.waitForTimeout( 2000 );
```

Comment:
As mentioned in the other comment, importing `NewOrder` page here is kinda confusing, could we define `waitForTimeout` in this file instead?

Reply:
@louwie17 Thanks for the suggestion. I implemented it in https://github.com/woocommerce/woocommerce-admin/pull/7646/commits/387c31aa44c58a845d2742b0492f35b36077ebe2